packages/flutter/test/material/text_field_test.dart

```diff
@@ -1018,6 +1018,82 @@ void main() {
     expect(controller.text, 'abc d${testValue}ef ghi');
   });
 
```

Comment:
Can you add a test in cupertino/text_field_test.dart as well since it's a different implementation?

Reply:
Thanks for your reminder!

The corresponding regression test logic has been added(this [commit](https://github.com/flutter/flutter/pull/29809/commits/8d990c991c84021de4856ae1a370f7e7dc01ba49)).